en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +66,86 @@ We will only consider merging post-beta breaking API changes if they are necessa
+- silverstripe/campaign-admin [![CI](https://github.com/silverstripe/silverstripe-campaign-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-campaign-admin/actions/workflows/ci.yml)
+- silverstripe/cms [![CI](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml)
+- silverstripe/elemental-banner-block [![CI](https://github.com/silverstripe/silverstripe-elemental-bannerblock/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-elemental-bannerblock/actions/workflows/ci.yml)
+- silverstripe/session-manager [![CI](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml)
+- silverstripe/recipe-ccl [![CI](https://github.com/silverstripe/recipe-ccl/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/recipe-ccl/actions/workflows/ci.yml)
+- silverstripe/recipe-content-blocks [![CI](https://github.com/silverstripe/recipe-content-blocks/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/recipe-content-blocks/actions/workflows/ci.yml)
+- silverstripe/recipe-core [![CI](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml)
+- silverstripe/staticpublishqueue [![CI](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml)
+
+#### Other issues
+
+`tractorcow/silverstripe-fluent` is temporarily being release under `silverstripe/silverstripe-fluent`.
+
+## Upgrading your third party module
+
+One of the primary purposes of the beta testing phase is to give third party maintainers and developers an opportunity to upgrade their modules to be compatible with Silverstripe CMS 5.
+
+### Planning your upgrade
+
+You should decide if you intend to do a new major release of your module to add support for Silverstripe CMS 5.
+
+Most of key Silverstripe CMS APIs have not change between Silverstripe CMS 4 and Silverstripe CMS 5. Take time to read this changelog to identify changes that might be problematic for your project.
+
+Depending on how complex your module is, it may be completely practical to create a version that works with both Silverstripe CMS 4 and Silverstripe CMS 5. We do not necessarily recommend this approach. But you may wish to consider it to minimise your maintenance burden.
+
+If you decide that you want to maintain distinct Silverstripe CMS 4 and Silverstripe CMS 5 versions of your module, then you should fork your current development branch.
```

Comment:
Why are we saying for them to create a new fork? What do you mean by that?

Reply:
If you just meant they should have a new branch for the CMS 5 compatible version which is separate from the current cms 4 compatible branch, it's probably best to just explicitly state that. "Fork" in github is its own very specific concept which is more about repositories than it is about branches.